class_exercises/warmup.js

```diff
@@ -0,0 +1,70 @@
+Warm Up Week 2, Day 2
+Let’s declare some variables:
+- Define variable named pet, which is a *string* of the name of one of your pets (or the pet you wish you had)
+- Define variable named toes, which is an *integer* of the number of toes you have
+- Define a variable named tired, which is a *boolean* value of whether you feel tired or not right now!
+Let’s create some arrays:
+- Define a variable named fruits, which is an *array* of three of your favorite fruits
+- Define a variable named friends, which is a *multi-dimensional array*, including the names of three friends and their favorite color (make it up!)
+Let’s create some objects:
+- Define a variable named person. Set it equal to an *object* with the attributes name, toeCount, and hairColor.
+- Log to the console:
```

Comment:
Make sure to comment out the non-code text in a javascript file. You won't be able to run the code with uncommented plain text in the file, so it will be difficult for you to test your own work.